php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2913,6 +2914,21 @@ abstract class NDB_BVL_Instrument extends NDB_Page
             $this->freeze();
         }
 
+        if ($success) {
+            $sessionID = $request->getQueryParams()["sessionID"];
+            $candID    = $request->getQueryParams()["candID"];
+            $baseURL   = \NDB_Factory::singleton()->settings()->getBaseURL();
+            $pageURL   = !empty($this->page) ? urlencode($this->page) ."/" : "";
+            $url       = $baseURL . "/instruments/" .
+                urlencode($this->testName) . "/" .
+                $pageURL .
+                '?commentID=' . urlencode($this->getCommentID()) .
+                '&candID=' . urlencode($candID) .
+                '&sessionID=' . urlencode($sessionID);
+            return (new \LORIS\Http\Response())
+                ->withHeader("Location", $url);
```

Comment:
The response should also have a 303 See Other response code.

Reply:
what other response code ?

Reply:
No other one. The code's name is `See Other`.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/303

Reply:
line 2846 above in the same file has the exact same code, should we move this to a function ?

Reply:
2846 already had the 303? I don't see why it would be moved to a function.. setting the response code is always going to be context specific

Reply:
dont like code duplication